DateTimeField returns DatetimeWithNanoseconds instead of datetime

Following https://github.com/orijtech/spanner-orm/pull/83:
```
======================================================================
FAIL: test_for_datetimefields_saves_as_much_precision_as_was_given (basic.tests.ModelInstanceCreationTests)
as much precision in *seconds*
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tim/code/django/tests/basic/tests.py", line 98, in test_for_datetimefields_saves_as_much_precision_as_was_given
    self.assertEqual(Article.objects.get(id__exact=a1.id).pub_date, datetime(2005, 7, 31, 12, 30))
AssertionError: DatetimeWithNanoseconds(2005, 7, 31, 12, 30, tzinfo=<UTC>) != datetime.datetime(2005, 7, 31, 12, 30)
```
`DatabaseOperations.convert_datetimefield_value()` (as I commented on the PR) isn't correct.

--- a/spanner/django/operations.py
+++ b/spanner/django/operations.py
@@ -1,3 +1,5 @@
+from datetime import datetime
+
 from django.conf import settings
 from django.db.backends.base.operations import BaseDatabaseOperations
 from django.utils import timezone
@@ -53,6 +55,13 @@ class DatabaseOperations(BaseDatabaseOperations):
     def convert_datetimefield_value(self, value, expression, connection):
         if value is None:
             return value
-        if timezone.is_aware(value):
-            return value
-        return timezone.make_aware(value, self.connection.timezone)
+        # Cloud Spanner returns the
+        # google.api_core.datetime_helpers.DatetimeWithNanoseconds subclass
+        # of datetime with tzinfo=UTC (which should be replaced with the
+        # connection's timezone). Django doesn't support nanoseconds so that
+        # part is ignored.
+        return datetime(
+            value.year, value.month, value.day,
+            value.hour, value.minute, value.second, value.microsecond,
+            self.connection.timezone,
+        )

